lua/nvim-treesitter.lua

```diff
@@ -15,5 +15,6 @@ end
 -- To install, run `:lua require'nvim-treesitter'.install_parser('language')`
 -- we should add a vim layer over the lua function
 M.install_parser = install.install_parser
+M.list_parsers = install.list_parsers
```

Comment:
this function is not yet included, you should merge my list_parsers PR onto yours or in master first.

Reply:
Actually I used your PR as a base for this one, but, given the modifications, this line make no sense anymore, you are right